heroku/cli
# migrate(W-14169298): spaces: Upgrade spaces:peering:accept

This PR moves `spaces:peering:accept" to the CLI and migrates to TS

### Test instructions
1. checkout this branch and build
2. run `./bin/run spaces:peering:accept YOUR_PCX_ID YOUR_SPACE` or `./bin/run spaces:peering:accept --pcxid YOUR_PCX_ID --space YOUR_SPACE`
3. Verify the output is the same as the original.
